### ISSUE TITLE
mapping between schemas in different yago versions

### ISSUE THREAD

**Issue by wwymak** [yago-naga/yago4#20]
Hi, is there a map of the old yago schema to the current schema.org ones? E.g. in yago 3, a person will have `rdf:type` of `yago:wordnet_person_100007846` but in yago4 it's `schema:Person` . I am trying to compare the entries for entites across the versions and it would be nice if there's a mapping that I can use.

**Comment by fmsuchanek** [comment]
I totally agree, but we do not have such a mapping unfortunately... That said, the classes of YAGO-3 are WordNet, and the classes of YAGO-4 are schema.org, so there may be other projects that produced such a mapping...
